src/eureka/S1_detector_processing/ramp_fitting.py

```diff
@@ -369,6 +369,8 @@ def calc_opt_sums_uniform_weight(rn_sect, gain_sect, data_masked, mask_2d,
 
     Parameters
     ----------
+    ramp_data : RampData
+        Input data necessary for computing ramp fitting. (Unused)
```

Comment:
Ok, you've got me stumped.  Why are you including a variable that goes unused?

Reply:
Because their function now has the `ramp_data` as a required input (see [here](https://github.com/spacetelescope/stcal/blob/a57fe3474291b855d937e920ce8ebfbced4b990d/src/stcal/ramp_fitting/ols_fit.py#L3833)), so without making this change we get a crash because of mismatched arguments